3412.xml

```diff
@@ -0,0 +1,288 @@
+				<Operations>RW</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description>Maximum number of times a message should be repeated within a mesh network.</Description>
+			</Item>
+			<Item ID="17">
+				<Name>Number of repeats</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description>Number of mesh nodes between the device and the destination device (e.g. another device or a router) including the destination node, on a mesh network.</Description>
+			</Item>
+			<Item ID="18">
+				<Name>Signal to noise ratio</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units>dB</Units>
+				<Description>Ratio of signal power to the noise power.</Description>
+			</Item>
+			<Item ID="19">
+				<Name>Communication failure</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Boolean</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description>Set to True when the device can't communicate properly.</Description>
+			</Item>
+			<Item ID="20">
+				<Name>Received Signal Strength Indication</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units>dBm</Units>
+				<Description>Signal strength of the communication network measured by the device (a.k.a. RSSI or signal level).</Description>
+			</Item>
+			<Item ID="21">
+				<Name>IMSI</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>String</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description>Device’s International Mobile Subscriber Identity.</Description>
+			</Item>
+			<Item ID="22">
+				<Name>IMEI</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>String</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description>Device’s International Mobile Equipment Identity.</Description>
+			</Item>
+			<Item ID="23">
+				<Name>Current Communication Operator</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>String</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description>Device’s current communication operator.</Description>
```

Comment:
Is this supposed to be machine-readable? If yes, would be important to define the format (preferably referencing a standard)